src/test/java/no/regnskap/mapper/MapperTest.java

```diff
@@ -0,0 +1,141 @@
+package no.regnskap.mapper;
+
+import no.regnskap.TestData;
+import no.regnskap.generated.model.Regnskap;
+import no.regnskap.model.RegnskapDB;
+import no.regnskap.model.RegnskapFieldsDB;
+import no.regnskap.model.RegnskapXmlWrap;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static no.regnskap.mapper.RegnskapXmlMapperKt.deserializeXmlString;
+import static no.regnskap.mapper.RegnskapMapperKt.mapXmlListForPersistence;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MapperTest {
+
+    private static RegnskapXmlWrap xmlWrap;
+    private static List<RegnskapDB> listRegnskapDB;
+    private static List<Regnskap> listRegnskap;
+
+    @BeforeClass
+    public static void mappingFromXmlToResponse() throws IOException {
+        xmlWrap = deserializeXmlString(TestData.xmlTestString);
+        listRegnskapDB = mapXmlListForPersistence(xmlWrap.getList());
+
+        listRegnskap = listRegnskapDB
+            .stream()
+            .map(RegnskapMapperKt::mapPersistenceToGenerated)
+            .collect(Collectors.toList());
+    }
+
+    @Test
+    public void allFieldsAreMapped() {
+        Assert.assertEquals(44, xmlWrap.getList().get(0).getPosts().size());
+        Assert.assertEquals(60, xmlWrap.getList().get(1).getPosts().size());
+        Assert.assertEquals(40, xmlWrap.getList().get(2).getPosts().size());
+        Assert.assertEquals(45, xmlWrap.getList().get(3).getPosts().size());
+    }
+
+    @Test
+    public void equalOrgnrAndRegnaarIsMerged() {
+        Assert.assertEquals(4, xmlWrap.getList().size());
+        Assert.assertEquals(xmlWrap.getList().get(0).getHead().getOrgnr(),xmlWrap.getList().get(1).getHead().getOrgnr() );
```

Comment:
I seldom care about whitespace, but right here it might be good to have a space after comma to visually separate parameters